config/routes.rb

```diff
@@ -1,15 +1,16 @@
 Rails.application.routes.draw do
   root to: "home#index"
 
-  get "/meal-plan" => "meals#show"
-  get 'auth/:provider/callback', to: 'sessions#create'
-  get '/auth/failure' do
-	  flash[:notice] = params[:message]
-	  redirect '/'
-	end
+  get 'auth/:provider/callback', to: 'sessions#create', via: :all
+  match 'signout', to: 'sessions#destroy', via: :all
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.